runtime/vm/JFRConstantPoolTypes.cpp

```diff
@@ -1160,8 +1161,28 @@ VM_JFRConstantPoolTypes::addClassLoadingStatisticsEntry(J9JFRClassLoadingStatist
 	entry->loadedClassCount = classLoadingStatisticsData->loadedClassCount;
 	entry->unloadedClassCount = classLoadingStatisticsData->unloadedClassCount;
 
-	index = _classLoadingStatisticsCount;
+	index = (U_32)_classLoadingStatisticsCount;
 	_classLoadingStatisticsCount += 1;
```

Comment:
I don't think this cast is needed.
```suggestion
	index = _classLoadingStatisticsCount;
```

Reply:
Yes this line is supposed to belong to another PR...